doc/ocaml-ecosystem.rst

```diff
@@ -0,0 +1,89 @@
+The OCaml Ecosystem
+===================
+
+The OCaml ecosystem is not monolithic: the compiler and tools are not
+maintained by the same entities. As such, it can be difficult to understand the
+history and roles of the various pieces of this ecosystem. The goal of this
+page is to give a quick overview of the situation and of the role that Dune
+plays in it.
+
+The Compiler Distribution: Compiling and Linking
+------------------------------------------------
+
+The compiler distribution contains "core" tools including the compilers
+(``ocamlc`` and ``ocamlopt``). They turn source files (with extensions ``.ml``
+and ``.mli``) into executables and libraries. Dependencies between compiled
+objects only exist at the module level, so this is a low-level tool.
+
+Findlib: Metadata for libraries
+-------------------------------
+
+Findlib is a tool that defines the concept of library, so that libraries can
+depend on other libraries on top of the notion of module. Definitions of
+libraries, and other pieces of metadata, are stored in ``META`` files.
+
+Findlib ships an executable named ``ocamlfind`` that can be used as a wrapper
+on top of the compilers to perform tasks such as producing an executable from
+compiled object files and external libraries.
+
+Opam: a Collection of Software Projects
+---------------------------------------
+
+Opam is a package manager. It is used to determine which packages are
+necessary, and how to fetch and build them. Packages can contain libraries,
+executables, and other kinds of files.
+
+The notion of version is specific to opam. If your project uses a function
```

Comment:
```suggestion
The notion of version is specific to ``opam``. If your project uses a function
```

Reply:
Or Opam. There are a few other mentions below too.

Reply:
according to our style guide (which I'll embed a copy in the docs), it's lowercase opam, no monospace, as confirmed by the authors.